scripts/examples/webos_example.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+#
+#    Copyright (c) 2021 Project CHIP Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+# Download webOS OSE NDK
+echo "##### Download webOS OSE NDK #####"
+wget https://sourceforge.net/projects/webos-iotivity-patches/files/webos-sdk-x86_64-cortexa7t2hf-neon-vfpv4-toolchain-2.14.1.g.sh
+wget https://sourceforge.net/projects/webos-iotivity-patches/files/webos-sdk-x86_64-cortexa7t2hf-neon-vfpv4-toolchain-2.14.1.g.host.manifest
+wget https://sourceforge.net/projects/webos-iotivity-patches/files/webos-sdk-x86_64-cortexa7t2hf-neon-vfpv4-toolchain-2.14.1.g.target.manifest
+wget https://sourceforge.net/projects/webos-iotivity-patches/files/webos-sdk-x86_64-cortexa7t2hf-neon-vfpv4-toolchain-2.14.1.g.testdata.json
+
+# Grant execute permission for NDK install script
+chmod 555 webos-sdk-x86_64-cortexa7t2hf-neon-vfpv4-toolchain-2.14.1.g.sh
+
+# Install webOS OSE NDK
+echo "##### Install webOS OSE NDK #####"
+sudo ./webos-sdk-x86_64-cortexa7t2hf-neon-vfpv4-toolchain-2.14.1.g.sh
```

Comment:
why is sudo needed here?

I do not trust scripts to "downloads something off the internet and executes as root".

Reply:
Looking at the script, it seems to support `-d` for target_sdk_dir. Lets use that and not root.

Reply:
We should also check and download only if the sdk does not exist. The SDK itself seems very large: 2.1GB.

Downloading every compilation would be extreamly slow.

Reply:
+100 @andy31415 -> Usage of a public git repo is more preferable than a wget scripting

Reply:
> why is sudo needed here?
> 
> I do not trust scripts to "download something off the internet and execute as root".

Yes this is totally sketchy !

Reply:
If you guys really want to go this way then you will have to verify a legit signature of the package... (SHA, MD5) 
Once again - @cabin15, it would be better to use the common scheme of external repositories being located into the third_party folder as a git submodule

Reply:
@andy31415, @jmeg-sfy 
Thank you for your review.
According to your review, I stopped using 'wget'. And I registered the webOS SDK resources as a submodule located under third_party.
Would this be fine for the baseline start?
Thank you.